src/app/trainingdescription/trainingdescription.component.ts

```diff
@@ -2,23 +2,26 @@ import { Component, OnInit } from '@angular/core';
 import { InternshipsService } from '../core/internships.service';
 import { Internship } from '../types';
 
-
 @Component({
   selector: 'ia-trainingdescription',
   templateUrl: './trainingdescription.component.html',
   styleUrls: ['./trainingdescription.component.scss']
 })
 export class TrainingdescriptionComponent implements OnInit {
 
-  internships!: Internship[];
+  internships: Internship[] = [];
+  duration: string = "";
+  difference: number = 0;
 
   constructor(private internshipsService: InternshipsService) { }
 
   ngOnInit(): void {
     this.internshipsService.getInternshipList().subscribe(internships => {
```

Comment:
As we discussed today don't fetch the entire list of internships here. Get one, its id can be hardcoded for now